Classes/Service/ConfigurationService.php

```diff
@@ -325,7 +325,6 @@ public function getContentElementFormInstances()
             } else {
                 $extensionKey = $registeredExtensionKey;
             }
-            $extensionKey = ExtensionNamingUtility::getExtensionKey($extensionKey);
```

Comment:
Can you describe what this line solves? It would cause the Provider instance to be configured differently - and carry a `Vendor.ExtensionName` format instead of `extension_key`.

Reply:
It seems flux for 9.2 wants a full qualified vendor.extension in the provider class.